ferrocene/doc/sphinx-shared-resources/README.md

```diff
@@ -6,7 +6,7 @@
 This repository contains the shared Sphinx resources we rely on for our
 Sphinx documentation websites, rather than duplicating them into each site.
 
-🚨 **The contents of this repository are strictly meant for Ferrocene use** 🚨  
```

Comment:
This white space is a markdown newline, it must not be removed.

Reply:
Ohhhhhh ... that's ... cool

Reply:
Or can I replace it with an actual newline?

Reply:
I added an actual newline